marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/CreateGranularPrivilegesCommand.java

```diff
@@ -117,10 +115,11 @@ public Integer getUndoSortOrder() {
 
     @Override
     public void execute(CommandContext context) {
-        if (new Versions(hubConfig).isVersionCompatibleWith520Roles()) {
+        if (new Versions(hubConfig).getMarkLogicVersion().isVersionCompatibleWith520Roles()) {
```

Comment:
The less we use Versions, the better - it's a confusing class because of its Spring dependencies. 

I think this should just be "new MarkLogicVersion(hubConfig.getManageClient()).isVersionCompatiableWith520Roles()". 

Essentially, all the actual logic in Versions for parsing version numbers should move into MarkLogicVersion, such that Versions becomes this thin class that is rarely used and only serves to create a MarkLogicVersion object.

Reply:
The Versions class still has `getMarkLogicVersionString()` (renamed from `getMarkLogicVersion()`) method which performs an eval using hubClient or hubConfig depending on what is present. 
This method is useful for cases where the user does not have access to "manage".